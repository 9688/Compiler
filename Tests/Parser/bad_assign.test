a =;
a = b =;
 = 7 + 2;