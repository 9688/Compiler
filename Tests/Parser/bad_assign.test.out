Ошибка в строке 1 позиции 3: требуется выражение
Ошибка в строке 2 позиции 7: требуется выражение
Ошибка в строке 3 позиции 8: требуется выражение
"ROOT"
